Lib/AspNet/Microsoft.WindowsAzure.Storage/project.json

```diff
@@ -1,91 +1,74 @@
 ﻿{
-    "version": "7.0.1.0",
+    "version": "7.1.1.0",
     "authors": [ "Microsoft Corporation" ],
     "description": "Azure Storage SDK for CoreCLR Preview",
     "dependencies": {
         "Microsoft.Data.OData": "5.6.4",
         "Microsoft.Data.Services.Client": "5.6.4",
         "System.Spatial": "5.6.4",
-        "Newtonsoft.Json":  "6.0.8"
+        "Newtonsoft.Json": "6.0.8"
     },
+
     "frameworks": {
-        "dotnet5.4": {
+        "netstandard1.6": {
+            "dependencies": {
+                "NETStandard.Library": "1.6.0",
+                "System.Dynamic.Runtime": "4.0.11",
+                "System.Security.Cryptography.Algorithms": "4.2.0"
+            },
+            "imports": [
+                "dnxcore50",
+                "portable-net451+win8"
+            ]
+        },
+        "netcoreapp1.0": {
```

Comment:
Curious ... Why `netcoreapp1.0` here? ... This lib isn't a runnable app, is it? ... or is it? lol :smile: